File: lib/discord/interactions/commands/reopen.dart
import "package:codercord/discord/interactions/commands/close.dart";
import "package:codercord/values.dart";

import "package:nyxx_interactions/nyxx_interactions.dart";

SlashCommandBuilder getCommand() {
  return SlashCommandBuilder(
    "reopen",
    "Reopens your post",
    [],
    guild: coderServer.id,
    canBeUsedInDm: false,
  )..registerHandler((p0) async {
      await handleIssueStateCommand(p0, false);
    });
}

File: src/commands/util/reopen.ts
import { handleIssueStateCommand } from "./close.js";

import {
  SlashCommandBuilder,
  type ChatInputCommandInteraction,
} from "discord.js";

export default {
  data: new SlashCommandBuilder()
    .setName("reopen")
    .setDescription("Reopens your issue"),

  execute: (interaction: ChatInputCommandInteraction) =>
    handleIssueStateCommand(interaction, false, false),
};
